boards/arm/olimex_stm32_e407/doc/olimex_stm32_e407.rst

```diff
@@ -0,0 +1,328 @@
++-------+--------------+-------+--------------+
+| 7     | PE4/D3       | 17    | PE14         |
++-------+--------------+-------+--------------+
+| 8     | PE5/D4       | 18    | PE15         |
++-------+--------------+-------+--------------+
+| 9     | PE6/D5       | 19    | +5V          |
++-------+--------------+-------+--------------+
+| 10    | PE7          | 20    | GND          |
++-------+--------------+-------+--------------+
+
+PF
+
++-------+--------------+-------+--------------+
+| PIN # | Signal Name  | Pin # | Signal Name  |
++=======+==============+=======+==============+
+| 1     | +3.3V        | 11    | PF8/A3       |
++-------+--------------+-------+--------------+
+| 2     | GND          | 12    | PF9/A4       |
++-------+--------------+-------+--------------+
+| 3     | PF0          | 13    | PF10/A5      |
++-------+--------------+-------+--------------+
+| 4     | PF1          | 14    | PF11/A6      |
++-------+--------------+-------+--------------+
+| 5     | PF2          | 15    | PF12         |
++-------+--------------+-------+--------------+
+| 6     | PF3          | 16    | PF13         |
++-------+--------------+-------+--------------+
+| 7     | PF4          | 17    | PF14         |
++-------+--------------+-------+--------------+
+| 8     | PF5          | 18    | PF15         |
++-------+--------------+-------+--------------+
+| 9     | PF6/A1       | 19    | +5V          |
++-------+--------------+-------+--------------+
+| 10    | PF7/A2       | 20    | GND          |
++-------+--------------+-------+--------------+
+
+PG
+
++-------+--------------+-------+--------------+
+| PIN # | Signal Name  | Pin # | Signal Name  |
++=======+==============+=======+==============+
+| 1     | +3.3V        | 11    | PG8/D7       |
++-------+--------------+-------+--------------+
+| 2     | GND          | 12    | PG9          |
++-------+--------------+-------+--------------+
+| 3     | PG0          | 13    | PG10/UEXT_CS |
++-------+--------------+-------+--------------+
+| 4     | PG1          | 14    | PG11/TX_EN   |
++-------+--------------+-------+--------------+
+| 5     | PG2          | 15    | PG12/D8      |
++-------+--------------+-------+--------------+
+| 6     | PG3          | 16    | PG13/TXD0    |
++-------+--------------+-------+--------------+
+| 7     | PG4          | 17    | PG14/TXD1    |
++-------+--------------+-------+--------------+
+| 8     | PG5          | 18    | PG15/D9      |
++-------+--------------+-------+--------------+
+| 9     | PG6          | 19    | +5V          |
++-------+--------------+-------+--------------+
+| 10    | PG7/D6       | 20    | GND          |
++-------+--------------+-------+--------------+
+
+System Clock
+============
+
+OLIMEX-STM32-E407 has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 12 MHz. The processor can setup HSE to drive the master clock,
+which can be set as high as 168 MHz.
+
```

Comment:
OK understood what you mean, I will have to see if I can get openocd to work, cause until now I just used the ST-Link program to flash the hex/bin files. 

I did test the blinky, button, hello_world and http_server samples (the http_server is ofcourse not functional due to missing ethernet driver).

I can/will update the docu, but it will take a few days before I have time to try out openocd. 

- Erwin